frame/instrumental-strategy-pablo/src/tests.rs

```diff
@@ -88,3 +89,173 @@ fn rebalance_emits_event() {
 		}));
 	});
 }
+
+// -------------------------------------------------------------------------------------------------
+//                                             Set pool_id for asset
```

Comment:
NIT: Love the organization but use modules to codify this
i.e
```rust
#[cfg(test)]
mod set_pool_id_for_asset {
// test go here...
}
```

Reply:
Did it. Thanks!